browser/css/toolbar.css

```diff
@@ -1727,11 +1727,13 @@ menu-entry-with-icon.padding-left + menu-entry-icon.width */
 
 #follow-editor {
 	padding: 8px;
+	display: flex;
+	align-items: center;
 }
 
 #follow-editor .follow-editor-checkbox {
-	margin-right: 12px;
-	margin-left: 7px;
+	margin-right: 11px;
+	margin-left: 6px;
```

Comment:
wouldn't it be better to drop the margin-left? I think we can do that by making sure that the checkbox has the same size as the avatars (28px). Lastly, it would be good to avoid using -left or -right and instead use margin-inline-start/end  in this case https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline-end this way we are sure that the margin will be correctly applied for property depending on the values defined for [writing-mode](https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode), [direction](https://developer.mozilla.org/en-US/docs/Web/CSS/direction), and [text-orientation](https://developer.mozilla.org/en-US/docs/Web/CSS/text-orientation).

Reply:
![Screenshot from 2024-12-20 12-47-08](https://github.com/user-attachments/assets/1636abd2-6fb7-4df0-9a8a-172dfb24af1a)

Using margin-inline-end/start is better, but adjusting the checkbox to 28px is not better for UX

Reply:
agreed, I thought somehow that the check box itself would not change the size that would only occupy more space but that graphically would be exact the same perceived. you are correct, thanks